src/main/java/org/csanchez/jenkins/plugins/kubernetes/PodTemplate.java

```diff
@@ -483,7 +483,7 @@ public Long getRunAsGroupAsLong() {
 
     @DataBoundSetter
     public void setSupplementalGroups(String supplementalGroups) {
-        this.supplementalGroups = supplementalGroups;
+        this.supplementalGroups = Util.fixEmpty(supplementalGroups);
```

Comment:
Ought to fix a `Snippetizer` bug anyway, right?

Reply:
I think `Snippetizer` had it right already - https://github.com/jenkinsci/kubernetes-plugin/blob/529e89b07afb05375d1589d0383145738ea3948a/src/main/java/org/csanchez/jenkins/plugins/kubernetes/pipeline/PodTemplateStep.java#L381 but the unit test was setting it to empty string at https://github.com/jenkinsci/kubernetes-plugin/pull/805/files#diff-b71fe20fbec70935a4922665d36879a2R128